TouchControllerWS.h

```diff
@@ -1,14 +1,23 @@
 #include <FS.h>
-#include <Adafruit_STMPE610.h>
 
 #ifndef _TOUCH_CONTROLLERWSH_
 #define _TOUCH_CONTROLLERWSH_
 
+#ifdef TOUCH_CS
+  #include <Adafruit_STMPE610.h>
+#else  // use I2C
+  #include <Adafruit_TSC2007.h>
+#endif
```

Comment:
For some Arduino reason, the `#define` of `TOUCH_CS` in `settings.h` is not being seen here. So the logic always #includes the TSCC2007 header and the build fails at link time.

Suggest moving the `#define` of `TOUCH_CS` to this file and place immediately above this preproc logic. Can then add a comment in `settings.h` that this needs to be done for STMP. This worked with local testing.